rocAL/rocAL/source/api/rocal_api_data_transfer.cpp

```diff
@@ -35,7 +35,7 @@ rocalToTensor32(RocalContext p_context, float *out_ptr, RocalTensorLayout tensor
     auto context = static_cast<Context*>(p_context);
     try
     {
```

Comment:
Please change the braces of try catch block to google style if possible

Reply:
That will show more changes in this PR